test_1.spec.js

```diff
@@ -0,0 +1,41 @@
+import { test, expect } from '@playwright/test';
+
+test('Log In and Checking element', async ({ page }) => {
+
+  await page.goto('https://www.saucedemo.com/');
```

Comment:
Let's use [beforeEach](https://playwright.dev/docs/api/class-test#test-before-each-1) function to perform login

Reply:
Fixed